src/components/App/App.js

```diff
@@ -60,26 +68,29 @@ const App = () => {
 
   const handleNameSelect = useCallback(async ({ value: name }) => {
     await scrollToTop();
-    report('Filter', 'name', 'name');
+    report('Filter', 'name', name);
```

Comment:
This one is on purpose to avoid saving personal data on 3rd party (GDPR)

Reply:
reverted!